pom.xml

```diff
@@ -58,7 +58,7 @@
         <air.test.thread-count>4</air.test.thread-count>
         <air.test.jvmsize>1792m</air.test.jvmsize>
 
-        <air.javadoc.lint>-missing</air.javadoc.lint>
+        <air.javadoc.lint>--missing</air.javadoc.lint>
```

Comment:
This doesn't look right. This option is passed to the java compiler as an argument to the `-Xdoclint` setting. `--missing` is not a valid value.